saphir_macro/src/controller/handler.rs

```diff
@@ -64,7 +64,7 @@ impl ArgsReprType {
                 Err(Error::new_spanned(p, "Invalid option type"))
             }
             _params => Ok(ArgsReprType::Params {
-                is_query_param: !attrs.methods_paths.iter().any(|(_, path)| path.contains(name)),
+                is_query_param: !attrs.methods_paths.iter().any(|(_, path)| path.contains(&format!("<{}>", name))),
```

Comment:
@richerarc Technically this add an un-necessary allocation, but since it's only tested at server startup and writing a non-allocating check is not as trivial and readable as this solution I think this is fine?